src/abi.h

```diff
@@ -327,4 +327,41 @@ struct netplan_net_definition {
     /* netplan-feature: eswitch-mode */
     char* embedded_switch_mode;
     gboolean sriov_delay_virtual_functions_rebind;
+
+    /* vrf */
+    /* netplan-feature: vrf */
+    char* vrf;
+    guint vrf_table;
+
+    /* vxlan */
+    /* netplan-feature: vxlan */
+    guint vxlan_vni;
+    GArray* vxlans;
+    gboolean neigh_suppress;
```

Comment:
This seems to be a rather generic `[Bridge]` setting, not necessarily related to VXLANs.. Is it required for VXLAN? If so, we should implement it as a generic bridge feature (and probably use a tristate type, too).

Otherwise, I'd like to just drop it from this PR, as it seems unrelated. Especially, as there is currently no implementation in any backend (neither `networkd.c` nor `nm.c` for this `neigh_suppress` variable).

Reply:
Neighbor suppression (arp-nd-suppression) is required for VXLAN configuration. It is disabled by default, but many implementations, specifically those utilizing EVPN, will need it to be enabled. In systemd-networkd, it looks like this:

```bash
root@host1:~# cat /run/systemd/network/10-netplan-vxlan10.network 
[Match]
Name=vxlan10

[Link]
MTUBytes=8950

[Network]
LinkLocalAddressing=no
IPv6AcceptRA=no
ConfigureWithoutCarrier=yes
Bridge=bridge10

[Bridge]
NeighborSuppression=True
```

Reply:
The command to enable this feature is ```bridge link set dev vxlan10 neigh_suppress on```

Reply:
Thank you for your comments about this. Being a `[Bridge]` setting in sd-networkd, I still wonder if this is a setting that should be set on the VXLAN device itself, or rather on a bridge device that is a parent interface to the vxlan?

Reply:
It is a VXLAN setting (ie. you cannot enable it on the master Bridge interface). It may be confusing for netplan to have it as a bridge setting, especially since enslaving a VXLAN to a Bridge is not technically a requirement.

```bash
root@host1:~# bridge link set dev bridge10 neigh_suppress on
RTNETLINK answers: Operation not supported
```